howso/direct/core.py

```diff
@@ -1138,6 +1138,7 @@ def train(
         series: Optional[str] = None,
         session: Optional[str] = None,
         train_weights_only: bool = False,
+        skip_auto_analyze: bool = False,
```

Comment:
Same comment here about alphabetized keyword-only arguments. (don't forget the doc-string too).